### PR TITLE
Add support for dev server

### PR DESCRIPTION
**Pull request by fortuna**
You can now run the Outline Manager on the browser with a dev server:
```
yarn do server_manager/web_app/run
```

cc: @JonathanDCohen @mpmcroy

### CODE REVIEW

File: src/server_manager/base.webpack.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+const GENERATE_CSS_RTL_LOADER = path.resolve(__dirname, 'css-in-js-rtl-loader.js');
+
+exports.makeConfig = (options) => {
+  return {
+    mode: options.defaultMode,
+    entry: [
+      require.resolve('@webcomponents/webcomponentsjs/webcomponents-loader.js'),
+      path.resolve(__dirname, './web_app/ui_components/style.css'),
+      options.main,
+      // path.resolve(__dirname, './web_app/main.ts'),
```
Comment (alalamav) [inline src/server_manager/base.webpack.js:31]: nit: remove

File: src/server_manager/base.webpack.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+    ],
+    target: options.target,
+    devtool: 'inline-source-map',
+    // Run the dev server with `yarn workspace outline-manager run webpack-dev-server --open`
+    devServer: {
+      // contentBase: OUTPUT_BASE,
```
Comment (alalamav) [inline src/server_manager/base.webpack.js:37]: nit: remove

File: src/server_manager/base.webpack.js [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+    },
+    resolve: {extensions: ['.tsx', '.ts', '.js']},
+    plugins: [
+      new webpack.DefinePlugin({
+        // Hack to protect against @sentry/electron not having process.type defined.
+        'process.type': JSON.stringify('renderer'),
+        // Statically link the Roboto font, rather than link to fonts.googleapis.com
+        'window.polymerSkipLoadingFontRoboto': JSON.stringify(true),
+      }),
+      new webpack.IgnorePlugin(/^electron$/),
```
Comment (alalamav) [inline src/server_manager/base.webpack.js:76]: Is this related to the sentry workaround? Maybe add a comment explaining why we need it.

File: src/server_manager/base.webpack.js (hunk 1 above)
Comment (fortuna) [reply src/server_manager/base.webpack.js]: Removed

File: src/server_manager/base.webpack.js (hunk 2 above)
Comment (fortuna) [reply src/server_manager/base.webpack.js]: Removed

File: src/server_manager/base.webpack.js (hunk 3 above)
Comment (fortuna) [reply src/server_manager/base.webpack.js:74]: Yep. I added a comment.
